src/components/BBAnimated/BBAnimated.js

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from 'react'
+import './BBAnimated.scss'
+
```

Comment:
I'm thinking that if the functionality for this was from codePen we should credit Mike Dixon here too just to be safe.  If you wrote this, I'll be sure to learn it :-D

Reply:
This all was taken from codepen with small deletes of unnecessary code. I added author who created this component in our component too.

Reply:
thanks!